README.md

```diff
@@ -1,19 +1,27 @@
-# The manuscript bot for automated scholarly publishing
+# Python utilities for Manubot: Manuscripts, open & automated
```

Comment:
manubot.org uses `and` instead of `&` in the tagline.  Maybe that causes a line wrap though?